examples/node-groups/self-managed-node-groups/main.tf

```diff
@@ -27,6 +27,16 @@ locals {
     Blueprint  = local.name
     GithubRepo = "github.com/aws-ia/terraform-aws-eks-blueprints"
   }
+
+  ca_expander          = "priority"
```

Comment:
same for this example, lets just put directly where used since its only one location

Reply:
done